PVTPackage/source/MultiphaseSystem/DeadOilMultiphaseSystem.hpp

```diff
@@ -40,6 +40,7 @@ class DeadOilMultiphaseSystem final : public MultiphaseSystem, private TableRead
 
 private:
 
+  // Three-phase dead-oil 
```

Comment:
Nice doc. Could you use the Doxygen notations? (just the `@brief` though).

Reply:
Done